src/components/List.js

```diff
@@ -1,14 +1,27 @@
 import React from 'react';
+import { useHistory } from 'react-router-dom';
 import useFirestore from '../hooks/useFirestore';
 
 const List = ({ token }) => {
   const { docs } = useFirestore(token);
+  const history = useHistory();
+
   return (
     <div>
       <h1>List</h1>
+      {docs.length === 0 && (
+        <div>
```

Comment:
If you wanted, you could use a `section` tag instead of a div, or just use a React.Fragment. Might make sense to have a container around the paragraph and button, though.